docs/yaml-config.rst

```diff
@@ -144,6 +144,56 @@ documentation.
 
 		conf_file: project2/docs/conf.py
 
+python.extra_requirements
+`````````````````````````
+
+* Default: ``[]``
+* Type: List
+
+List of `extra requirements`_ sections to install, additionnaly to the
+`package default dependencies`_. Only works if `python.pip_install` option
+above is set to `True`.
```

Comment:
That one is on purpose, as I want it to link to the Python's constant. Like it's done all over the page for `None` and `False`.

See for instance the behaviour of single-tick-quoted keywords at: https://docs.readthedocs.io/en/latest/yaml-config.html#python-pip-install . Go there and click on `False`: you're redirected to Python's doc.